client/frontend/src/Videos/VideoExpansionPanel.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
```

Comment:
We are not using this page I dont believe, I might be mistaken, so can you delete the file?